### ISSUE TITLE
Docker-compose failed

### ISSUE THREAD

**Issue by weinrank** [pboehm/ddns#15]
OS: Ubuntu 19.04

How to reproduce:
* Checkout Repository
* rename and modify override yml file
* call `docker-compose --project-name ddns up -d --build`

```
➜  docker git:(master) docker-compose --project-name ddns up -d --build
Building ddns
Step 1/8 : FROM golang:1.9-alpine3.7
 ---> 8d301df1eb85
Step 2/8 : RUN apk add --no-cache git
 ---> Using cache
 ---> 0e59388d886a
Step 3/8 : WORKDIR /go/src/github.com/pboehm/ddns
 ---> Using cache
 ---> 7bac33614b9f
Step 4/8 : COPY . .
 ---> ebb520dbfa8b
Step 5/8 : RUN go-wrapper download   # "go get -d -v ./..."
 ---> Running in 7ed3346f744f
+ exec go get -v -d
github.com/garyburd/redigo (download)
Fetching https://gopkg.in/gin-gonic/gin.v1?go-get=1
Parsing meta tags from https://gopkg.in/gin-gonic/gin.v1?go-get=1 (status code 200)
get "gopkg.in/gin-gonic/gin.v1": found meta tag get.metaImport{Prefix:"gopkg.in/gin-gonic/gin.v1", VCS:"git", RepoRoot:"https://gopkg.in/gin-gonic/gin.v1"} at https://gopkg.in/gin-gonic/gin.v1?go-get=1
gopkg.in/gin-gonic/gin.v1 (download)
package gopkg.in/gin-gonic/gin.v1: code in directory /go/src/gopkg.in/gin-gonic/gin.v1 expects import "github.com/gin-gonic/gin"
Fetching https://golang.org/x/sync/errgroup?go-get=1
Parsing meta tags from https://golang.org/x/sync/errgroup?go-get=1 (status code 200)
get "golang.org/x/sync/errgroup": found meta tag get.metaImport{Prefix:"golang.org/x/sync", VCS:"git", RepoRoot:"https://go.googlesource.com/sync"} at https://golang.org/x/sync/errgroup?go-get=1
get "golang.org/x/sync/errgroup": verifying non-authoritative meta tag
Fetching https://golang.org/x/sync?go-get=1
Parsing meta tags from https://golang.org/x/sync?go-get=1 (status code 200)
golang.org/x/sync (download)
ERROR: Service 'ddns' failed to build: The command '/bin/sh -c go-wrapper download   # "go get -d -v ./..."' returned a non-zero code: 1
```

**Comment by ntlvn** [comment]
Happens also on Ubuntu 18.04 @pboehm

**Comment by pboehm** [comment]
Sorry for my late reply, I was right in the process of moving to another home.

The problem is, that the web framework `gin` has changed its import URI. I've applied some fixes in #16 that you can try.

**Comment by pboehm** [comment]
Closing this as #16 has been merged.
